BraveWallet/WalletStrings.swift

```diff
@@ -2094,5 +2094,12 @@ extension Strings {
       value: "View the addresses of your permitted accounts (required)",
       comment: "A text displayed below the account address in new site connection confirmation step, in order to make sure users double check the account address they are going to allow the dapp to connect with."
     )
+    public static let settingsDefaultBaseCurrencyTitle = NSLocalizedString(
+      "wallet.settingsDefaultBaseCurrencyTitle",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Default base currency",
```

Comment:
Lets title-case this, as it looks weird.  Maybe also sneak in a title case on the "Clear nonce" reset message below?